testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -503,6 +503,43 @@ QUnit.test("dxSelectBox automatically scrolls to selected item on opening", func
     assert.ok($popupContent.offset().top + $popupContent.height() > $selectedItem.offset().top, "selected item is visible");
 });
 
+QUnit.test("dxSelectBox automatically scrolls to selected item on opening after item search", function(assert) {
+    var items = [];
+    for(var i = 0; i <= 100; i++) {
+        items.push(i);
+    }
+
+    var $selectBox = $("#selectBox").dxSelectBox({
+        items: items,
+        searchTimeout: 0,
+        searchEnabled: true
+    });
+
+    $($selectBox.find(toSelector(DX_DROP_DOWN_BUTTON))).trigger("dxclick");
+
+    this.clock.tick(TIME_TO_WAIT);
```

Comment:
Does this test work without `clock.tick`? You have zero search timeout. Try to run it without ticks.